### PR TITLE
add no build isolation

### PR DESCRIPTION
**Pull request by JamesRobertsonGames**
jsonschema-path 0.3.2 

**Destination channel:** defaults

### Links

- [Upstream repository](https://github.com/p1c2u/jsonschema-path)
- Relevant dependency PRs:
  - moto needs this dependency in main

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,18 +9,18 @@ source:
   sha256: 4d0dababf341e36e9b91a5fb2a3e3fd300b0150e7fe88df4e55cc8253c5a3989
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --ignore-installed
+  skip: True # [py<37]
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
  skip: True # [py<38]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,18 +9,18 @@ source:
...
     - pip
     - poetry-core >=1.0.0
-    - python >=3.8
+    - python
   run:
     - pathable >=0.4.1,<0.5.0
-    - python >=3.8.0
+    - python
     - pyyaml >=5.1
     - referencing >=0.28.0,<0.31.0
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - referencing >=0.28.0,<0.32.0
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -9,18 +9,18 @@ source:
...
     - requests >=2.31.0,<3.0.0
```
Comment (cbouss) [inline recipe/meta.yaml:26]:
```suggestion
  run_constrained:
    - requests >=2.31.0,<3.0.0
```

**Review by cbouss — COMMENTED** [review]
Can you make sure constraints are aligned with https://github.com/p1c2u/jsonschema-path/blob/0.3.2/pyproject.toml#L43-L48 and https://github.com/p1c2u/jsonschema-path/blob/0.3.2/pyproject.toml#L1-L3? Thank you

**Comment by cbouss** [comment]
Even though requests is marked as optional, the resulting package metadata has it required. In that case, we better keep it required as well.

**Review by cbouss — APPROVED** [review]

**Review by danpetry — APPROVED** [review]

**Pull request merged**